spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configuration-watcher/src/main/resources/application-bus-amqp.yaml

```diff
@@ -0,0 +1,23 @@
+spring:
```

Comment:
instead of having `activate.on-profile` - I've added these 2 files, one for each profile: `bus-amqp` and `bus-kafka` - this fixed the build here, but strangely enough - not in `main`. This this PR is entirely about a different thing - I am marking it as ready to review, for now.